torch/_dynamo/debug_utils.py

```diff
@@ -536,7 +536,13 @@ def same_two_models(gm, opt_gm, example_inputs, only_fwd=False):
         log.warning("Could not generate fp64 outputs")
         fp64_ref = None
 
-    res = run_fwd_maybe_bwd(opt_gm, example_inputs, only_fwd)
+    try:
+        res = run_fwd_maybe_bwd(opt_gm, example_inputs, only_fwd)
+    except Exception as e:
+        # This means that the the minified graph is bad/exposes a different problem.
+        # As we are checking accuracy here, lets log the exception and return True.
+        log.warning(f"Unable to run graph because {e}")
```

Comment:
worth giving a better error message.
Something like `"While minifying the program in accuracy minification mode, ran into a runtime exception which is likely an unrelated issue. Logging the exception and moving on. Exception: {e}"`